services/addons/tests/firmware_manager/upgrade_scheduler/test_upgrade_scheduler.py

```diff
@@ -0,0 +1,790 @@
+from unittest.mock import call, patch, MagicMock
+from collections import deque
+import services.addons.tests.firmware_manager.upgrade_scheduler.test_upgrade_scheduler_values as fmv

```

Comment:
I ran into some issues with this import when running pytest from the services directory - removing 'services' solved the issue. Does anyone else see this issue when running tests?

Reply:
I am also running into this issue. I think the `services.` prefix should be removed from these import statements.

Reply:
Sounds good. There is a chance my local environment changed something.

Reply:
Addressed

Reply:
This works for me now!